pinc/codepoint_validator.inc

```diff
@@ -1,5 +1,6 @@
 <?php
-include_once($relPath."Project.inc");
+include_once($relPath."CharSuites.inc"); // get_project_or_quiz()
```

Comment:
We don't need any includes.

Reply:
But we do. Right now this is working because something else is pulling in `CharSuites.inc`, and we should be explicit about it.